quantum/nodes/not-gate/not-gate.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const util = require('util');
+const snippets = require('../../snippets');
+const shell = require('../../python').PythonShell;
+
+module.exports = function(RED) {
+
+  function NotGateNode(config){
+    RED.nodes.createNode(this, config);
+    this.name = config.name;
+    this.qubits = [];
+    const node = this;
+
+    node.on("input", async function (msg, send, done) {
```

Comment:
'node.on' should be 'this.on' instead for consistency purposes across the nodes.

Reply:
hmmm..... ok..